RealmTasks Apple/RealmTasks iOS/CellPresenter.swift

```diff
@@ -31,36 +31,38 @@ class CellPresenter<Item: Object where Item: CellPresentable> {
         self.items = items
     }
 
-    func deleteItem(item: Item) {
+    func delete(item: Item) {
         viewController.uiWrite {
-            guard !(item as Object).invalidated, let index = items.indexOf(item) else {
+            guard !item.isInvalidated, let index = items.index(of: item) else {
                 return
             }
 
             items.realm?.delete(item)
-            viewController.tableView.deleteRowsAtIndexPaths([NSIndexPath(forRow: index, inSection: 0)], withRowAnimation: .Left)
+            viewController.tableView.deleteRows(at: [IndexPath(row: index, section: 0)], with: .left)
+            viewController.didUpdateList(reload: false)
```

Comment:
I think my previous comment on this was lost. Why was this call to `didUpdateList(...)` added here? Was there a bug in the Swift 2.3 version of this code? Or was this added by mistake?